src/Event/ResponseEvent.php

```diff
@@ -16,11 +15,10 @@ class ResponseEvent extends Event
 {
     /**
      * @param ResponseInterface $response
-     * @param string            $request_name
      */
-    public function __construct($response, $request_name)
+    public function __construct($response)
```

Comment:
Does the response usually contain the request? If not, I think it would be helpful to see have the request with the response. Since there's usually just a generic code.

Reply:
It should because the request will always be of type `Omnipay\Common\Message\ResponseInterface` which needs to implement the `getRequest` method to return a `RequestInterface` object.